scrypto/src/component/stubs.rs

```diff
@@ -383,6 +383,9 @@ extern_blueprint_internal! {
         fn cancel_primary_role_badge_withdraw_attempt(&mut self);
         fn cancel_recovery_role_badge_withdraw_attempt(&mut self);
         fn mint_recovery_badges(&mut self, non_fungible_local_ids: Vec<NonFungibleLocalId>) -> Bucket;
+        fn lock_recovery_fee(&mut self, amount: Decimal);
+        fn withdraw_recovery_fee(&mut self, amount: Decimal) -> Bucket;
+        fn contribute_recovery_fee(&mut self, bucket: Bucket);
```

Comment:
@lrubasze - there are new methods to consider regarding Fungible/NonFungibleBucket. :)

Reply:
Perhaps it would be good to make these methods accept a `FungibleBucket` now instead of needing a protocol update to change them later :)

Reply:
For now, rubin is only updating the language layer.

Reply:
> @lrubasze - there are new methods to consider regarding Fungible/NonFungibleBucket. :)

so happy to see it 😁